R/get_kr_comp.R

```diff
@@ -0,0 +1,115 @@
+#' obtain kr component
+#' @param tmb_data (`mmrm_tmb_data`)\cr produced by [h_mmrm_tmb_data()].
+#' @param theta numeric vector of the theta estimate.
+#' @details the function retuns a named list, $P$, $Q$ and $R$, which corresponds to the
```

Comment:
maybe good to double check that math can be included like this or whether we need `\eqn` or something like that